core/controllers/admin.py

```diff
@@ -1307,11 +1307,19 @@ def _generate_dummy_translation_opportunities(
                     subtopic_page_domain.SubtopicPage
                     .create_default_subtopic_page(1, topic_id))
             else:
-                skill = skill_services.skill_fetchers.get_skill_by_id(skill_id)
+                # Here we use MyPy ignore because skill_fetchers is dynamically
+                # added to skill_services module and not explicitly exported.
+                skill = skill_services.skill_fetchers.get_skill_by_id( # type: ignore[attr-defined]
```

Comment:
Actually I don't understand why we don't just import skill_fetchers in this file, and call skill_fetchers.get_skill_by_id(). Can we do that? Ditto below.